scripts/cybershake/queue_monitor.py

```diff
@@ -151,6 +153,49 @@ def update_tasks(
                     "Disappeared from squeue. Creating a new task.",
                 )
             )
+            # When job failed, we want to log metadata as well
+            # Assumes that when enter a jobid, only one row will be will returnerd
```

Comment:
Do no "assume" while it can be forced.
sacct -X will force it not show info for steps
example:
"sacct -X -P -n -j {} -u {} -S {} -E {} -o {}".format(job_id,username,start_date,end_date,sacct_format)